src/probabilistic_model/probabilistic_circuit/exporter/draw_io_expoter.py

```diff
@@ -0,0 +1,28 @@
+from N2G import drawio_diagram
+
+from probabilistic_model.probabilistic_circuit.nx.probabilistic_circuit import ProbabilisticCircuit
+
+
+class DrawIoExporter:
+
+    model: ProbabilisticCircuit
+
+    def __init__(self, model: ProbabilisticCircuit):
+        self.model = model
+
+    def export(self) -> drawio_diagram:
+        diagram = drawio_diagram()
+        diagram.add_diagram("Structure", width=1360, height=1864)
+        for node in self.model.nodes:
+            diagram.add_node(id=str(hash(node)), **node.draw_io_style())
+
+        for source, target in self.model.unweighted_edges:
```

Comment:
there is a calculation for the positions in pc.plot_structure. Perhaps use that one for positioning